src/main/java/io/hops/hopsworks/expat/migrations/projects/util/XAttrHelper.java

```diff
@@ -74,6 +82,20 @@ private static byte[] getXAttrInt(DistributedFileSystemOps udfso, String path, S
   }
   
   private static String getXAttrName(String namespace, String name) {
-    return namespace + name;
+    return namespace + "." + name;
+  }
+  
+  public static byte[] getXAttr(DistributedFileSystemOps udfso, String path, String namespace, String name)
+    throws XAttrException {
+    return getXAttrInt(udfso, path, namespace, name);
```

Comment:
why not just make the `getXAttrInt` public?

Reply:
fixed